src/NServiceBus.Gateway.AcceptanceTests/When_listening_on_wildcard_uri.cs

```diff
@@ -25,7 +25,30 @@ public EndpointWithWildCardUriAsDefault()
                 {
                     var gatewaySettings = c.Gateway();
 
-                    gatewaySettings.AddReceiveChannel("http://+:25699/");
+                    gatewaySettings.AddReceiveChannel("http://+:25601/WildcardA/");
+                });
+            }
+        }
+
+        [Test]
+        public void Should_not_throw_exception_if_wildcard_channel_is_not_default()
+        {
+            Assert.DoesNotThrowAsync(async () =>
+                await Scenario.Define<ScenarioContext>()
+                    .WithEndpoint<EndpointWithWildCardUriAndFullyQualifiedDefault>(e => e.When(b => Task.FromResult(0)))
```

Comment:
`e => e.When(b => Task.FromResult(0))` what's this?
if you want to make sure the endpoint starts without an exception, use `.Done(c => c.EndpointsStarted)` instead.